charts/prometheus/values.yaml

```
@@ -565,6 +565,9 @@ server:
   # Use hostPort
   # hostPort: 9090
 
+  # Use portName
+  # portName: prometheus
```

Please, do not comment out fields.

```suggestion
  portName: ""
```

---

👍